tests/network-tests/run-test-node-docker.sh

```diff
@@ -9,7 +9,7 @@ cd $SCRIPT_PATH
 
 # Location that will be mounted as the /data volume in containers
 # This is where the initial members and balances files and generated chainspec files will be located.
-DATA_PATH=${DATA_PATH:=$(pwd)/data}
+DATA_PATH=${DATA_PATH:=$PWD/data}
```

Comment:
The `Error opening spec file: No such file or directory (os error 2)` still occurs occasionally (in [CI checks](https://github.com/Joystream/joystream/runs/4332735960?check_suite_focus=true) and also on my local machine).

I wonder if it's not because we're mounting the chainspec to `/data` and in `docker-compose.yml` we're also mounting `chain-data` to `/data`. After switching from `-v ${DATA_PATH}:/data` to `-v ${DATA_PATH}:/spec` I didn't run into this issue anymore.

Reply:
Well spotted, this issue does seem to have popped up after using local volumes. Although on mac it seems docker-compose was behaving differently. Fixed in f31ba786c985fa784e38c4b4053d528ddbd8a3c3